web/views/component_helpers/button.ex

```diff
@@ -0,0 +1,24 @@
+defmodule Healthlocker.ComponentHelpers.Button do
```

Comment:
Ooh nice, @tonydaly, I didn't understand how hound would work before, but this is sweet